src/com/company/Main.java

```diff
@@ -12,14 +12,14 @@ public static void main(String[] args){
 
         String txtfile = args[0];
         File file = new File(txtfile);
-        checkforsupport(file);
-        checkforexistance(file);
+        checkForSupport(file);
+        checkForExistance(file);
         String searchword = args[1];
```

Comment:
Please check what will happen if args[1] does not contain anything. For example if user did not pass any argument what happens to the program?

Reply:
i have solved this with checking the args array and continue if required args are given else the execution is stopped